python/tests/toolchains/run_acceptance_test.py

```diff
@@ -14,23 +14,49 @@
 
 import os
 import subprocess
+import sys
+import tempfile
 import unittest
 
 
 class TestPythonVersion(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        os.chdir("%test_location%")
+        python_version_test_dirname = os.path.dirname(
+            os.path.realpath("python_version_test.py")
+        )
+        rules_python_path = os.path.normpath(
+            os.path.join(python_version_test_dirname, "..", "..", "..", "..")
+        )
+
+        is_windows = "win" in sys.platform
+        if is_windows:
+            newline = "\r\n"
+            os.environ["HOME"] = tempfile.mkdtemp()
+            os.environ["LocalAppData"] = tempfile.mkdtemp()
```

Comment:
Bazel requires these variables be set or some flags be explicitly provided. This is easier IMO